solutions/gold/usaco-921.mdx

```diff
@@ -161,14 +130,13 @@ int main() {
 ```
 
 </CPPSection>
+
 <JavaSection>
 
 ```java
 import java.io.*;
 import java.util.*;
 
-// this one uses a segment tree for the xor-ing instead of a BIT, but overall
-// it's the same philosophy
 public class CowLand {
 	private static class MinSegTree {
```

Comment:
this is like a really old segtree impl, i'm not sure if it's consistent w/ the module? if so just like wrap it w/ code snips or whatever